config/index.js

```diff
@@ -118,6 +118,12 @@ class Config {
     return 500
   }
 
+  getGithubAppConfig () {
+    const id = this.env.github_app_id
+    const privateKey = Buffer.from(this.env.github_app_pem, 'base64').toString('utf8')
```

Comment:
TODO: confirm these two env vars are in prod (can by copied from staging)

Reply:
rog

Reply:
added